src/pages/TodoList/TodoList.jsx

```diff
@@ -1,7 +1,67 @@
-import React from "react";
+import React, { useContext } from "react";
+import { useState, useEffect } from "react";
+import TodoItem from "../../components/TodoItem/TodoItem";
+import { apiBaseUrl } from "../../../config";
+import { StatusLengthContext } from "../../../StatusLengthContext";
 
-const TodoList = () => {
-  return <h2>AllCompletedUncompleted</h2>;
+const TodoList = ({ status }) => {
+  const [tasks, setTasks] = useState([]);
+  const [error, setError] = useState(null);
+  const { setAllLength, setCompletedLength, setUncompletedLength } =
+    useContext(StatusLengthContext);
+
+  const completedLength = tasks.filter((task) => task.isDone).length;
+  const uncompletedLength = tasks.filter((task) => !task.isDone).length;
```

Comment:
Move this inside the `useEffect` where the dependency is `tasks`. You are not using it out of that scope if I'm not mistaken.

Reply:
You are rigth.Done.